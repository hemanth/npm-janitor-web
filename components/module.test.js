import React from 'react'
import Module from './module'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  let module = {
    name: 'module',
    errors: [],
    warnings: [],
    suggestions: []
  }
  const tree = renderer.create(
    <Module module={module}/>
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
