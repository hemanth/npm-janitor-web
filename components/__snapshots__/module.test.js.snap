// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders invalid module 1`] = `
<span
  className="fVpfmz"
>
  <a
    className="igpMrq"
    href="https://npmjs.com/module"
    target="blank"
  >
    <div
      className="bAiiZt"
    >
      module
    </div>
  </a>
  <div>
    <div>
      <div>
         
        1
         
        error
      </div>
      <div>
        - 
        name is not allowed
      </div>
      <br />
    </div>
    
    
  </div>
  <div
    className="kUklBY"
  />
</span>
`;

exports[`renders valid module 1`] = `
<span
  className="cZhuiQ"
>
  <a
    className="igpMrq"
    href="https://npmjs.com/module"
    target="blank"
  >
    <div
      className="bAiiZt"
    >
      module
    </div>
  </a>
  <div>
    
    
    
  </div>
  <div
    className="huXapt"
  />
</span>
`;
